library/src/main/java/com/getstream/sdk/chat/viewmodel/CreateChannelViewModel.kt

```diff
@@ -0,0 +1,68 @@
+package com.getstream.sdk.chat.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.ViewModel
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.livedata.ChatDomain
+
+class CreateChannelViewModel(
+        private val chatDomain: ChatDomain = ChatDomain.instance(),
```

Comment:
`chatDomain` can be called just `domain`.  And about any names: `domain`, `client` etc.